ui/admin/app/controllers/scopes/scope/host-catalogs/index.js

```diff
@@ -0,0 +1,51 @@
+import Controller from '@ember/controller';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { debounce } from 'core/decorators/debounce';
+import { inject as service } from '@ember/service';
+
+export default class ScopesScopeHostCatalogsIndexController extends Controller {
+  // =services
+
+  @service intl;
+
+  // =attributes
+
+  queryParams = [
+    'search',
+    { types: { type: 'array' } },
+    { providers: { type: 'array' } },
```

Comment:
types, & providers left here can be cleaned up

Reply:
Woops, good catch totally missed this. Removed it